setup.cfg

```diff
@@ -35,12 +35,14 @@ setup_requires =
     pytest-runner
 install_requires =
     makefun>=1.5.0
+    typing_extensions;python_version>='3.6'
```

Comment:
could you please add a comment before this line explaining why this is needed even for latest python ?